provider/datagen/src/transform/uprops/mod.rs

```diff
@@ -2,9 +2,11 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
-//! `icu_datagen::uprops` contains implementations of the [`ICU4X`]
+//! This module contains implementations of the [`ICU4X`]
 //! [data provider] interface backed by TOML files exported by the
-//! ICU4C icuwriteuprops tool. Create a directory containing TOML files for
+//! ICU4C icuwriteuprops tool.
```

Comment:
Nit (here and elsewhere): I know this is pre-existing but the tool is named "icuexportdata" not "icuwriteuprops" (this was an older name for the tool when it was being drafted)